Project.toml

```diff
@@ -18,6 +18,7 @@ Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 DensityInterface = "b429d917-457f-4dbc-8f4c-0cc954292b1d"
 Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
I also added `Documenter` in order to build the documentation with GitHub Actions.